woocommerce/woocommerce-gutenberg-products-block
# CSS editor fixes for Cart and Checkout blocks

### Screenshots
#### Cart
_Before:_
![imatge](https://user-images.githubusercontent.com/3616980/76441302-78835e00-63bf-11ea-8bb1-57c231ed4e13.png)

_After:_
![imatge](https://user-images.githubusercontent.com/3616980/76440471-2f7eda00-63be-11ea-9916-6a0e9fa393c6.png)

#### Checkout
_Before:_
![imatge](https://user-images.githubusercontent.com/3616980/76438295-17f22200-63bb-11ea-97ff-fa1d8d23c13c.png)

_After:_
![imatge](https://user-images.githubusercontent.com/3616980/76438213-fe50da80-63ba-11ea-8d3c-6043b39607aa.png)

### How to test the changes in this Pull Request:

1. Open the _Cart_ block in the editor and verify buttons have the proper background color and text is properly aligned.
2. Open the _Checkout_ block in the editor and verify there isn't a white box on top of the _First name_ field border.
